lib/panel-league/garbage.js

```diff
@@ -1,5 +1,6 @@
 const JKISS31 = require('../jkiss');
 const {floodFill} = require('./util');
+const uuidV4 = require('uuid/v4');
```

Comment:
While there is no official guidelines about the order of imports, I suggest placing third party stuff first and internal imports second, separated by empty line.